hack/jenkins/cloud_shell_functional_tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
you would need to add this to also to 
https://github.com/medyagh/minikube/blob/9136127de65b2732a08d132954d09cb9aa90eba9/hack/jenkins/minikube_set_pending.sh#L34

and ensure the List of Jobs that get Commit Status is updated (since I dont see a commit status for cloud shell)

Reply:
Added the Jenkins job to job list